ibm/service/database/resource_ibm_database.go

```diff
@@ -2837,6 +2875,30 @@ func validateGroupHostFlavor(groupId string, resourceName string, group *Group)
 	return nil
 }
 
+func validateMultitenantMemoryCpu(groupId string, resourceName string, resourceDefaults *Group, group *Group) error {
+	// TODO: Replace this with resourceDefaults.Memory.CPUEnforcementRatioCeiling when it is fixed
+	cpuEnforcementRatioCeiling := 16384
+	// members := resourceDefaults.Members.Minimum
+
+	// if group.Members != nil && members < resourceDefaults.Members.Allocation {
+	// 	members = group.Members.Allocation
+	// }
+
+	if group.CPU != nil && group.CPU.Allocation > 2 {
+		return nil
+	}
+
+	if group.Memory.Allocation < resourceDefaults.Memory.Minimum {
+		return fmt.Errorf("We were unable to complete your request: group.memory requires a minimum of %d megabytes. Try again with valid values or contact support if the issue persists.", resourceDefaults.Memory.Minimum)
+	}
+
+	if group.CPU != nil && group.Memory.Allocation < cpuEnforcementRatioCeiling*resourceDefaults.Members.Minimum && group.Memory.Allocation > group.CPU.Allocation*resourceDefaults.Memory.CPUEnforcementRatioMb {
```

Comment:
`cpuEnforcementRatioCeiling*resourceDefaults.Members.Minimum` will work for provisioning, but a customer might have horizontally scaled, so we'd need you use `cpuEnforcementRatioCeiling*resourceDefaults.Members.allocationCount`, right?

Reply:
@bpeicher yup you're correct! Thank you for catching that!